charts/graylog/values.yaml

```
@@ -356,6 +356,7 @@ graylog:
   ## ref: https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#update-strategies
   ##
   updateStrategy: RollingUpdate
+  #maxUnavailable: 10%
```

This is still require feature gate to enable. 
Please add a document on this field e.g.
```suggestion
  ## Control the maximum number of Pods that can be unavailable during an update 
  ## ref: https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#maximum-unavailable-pods
  ##
  # maxUnavailable: 10%
```